envoy/common/optref.h

```diff
@@ -58,7 +58,7 @@ template <class T> struct OptRef {
   /**
    * @return true if the object has a value.
    */
-  bool has_value() const { return ptr_ != nullptr; }
+  bool has_value() const { return ptr_ != nullptr; } // // NOLINT(readability-identifier-naming)
```

Comment:
nit: you can remove extra `//`

Reply:
I understand why this is needed if optref.h must be modified, but another option is to just remove optref.h from this PR. My guess is you thought you needed to change it somehow, and then realized you didn't.

Reply:
got it. I think this is an unexpected change. Although it fixed a clang tidy problem.

Reply:
Right -- it often happens that new clang-tidy checks are added leaving some errors in existing files that are only cleaned up the next time that file is modified. However in this case I don't think you need to modify this file at all so you might as well make the PR smaller.